app-backup/casync/files/fix-selinux-build.patch

```diff
@@ -0,0 +1,126 @@
+From ab7ef83b5aef07af6e8628eeab353a3fb043fadd Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
+Date: Wed, 26 Jul 2017 09:52:40 -0400
+Subject: [PATCH 1/2] Fix build without selinux and refuse --with=selinux when
```

Comment:
Just to verify, there is no intended patch `2/2`?

Reply:
I got this from https://github.com/systemd/casync/commit/ca268c88cc225a33be569ccabab7292e71576935.patch

Since the PR has a second patch fixing a segfault, github generated a `1/2` subject, but I'm fine not picking up 2/2 until the next casync release since it's in an area of the code we don't use currently.